src/selectors/entities/posts.js

```diff
@@ -21,6 +43,99 @@ export const getPostsInCurrentChannel = createSelector(
     }
 );
 
+export function makeGetPostsInChannel() {
```

Comment:
This is great, @hmhealey maybe after this goes in you can replace the one we have in RN in `channel_post_list` with this one and have it only in one place ;)